keyboards/handwired/d48/config.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Andrew Dunai
+#define PRODUCT         D48
+#define DESCRIPTION     Proton C based 12x4 ortholinear keyboard with 2 encoders, 128x32 OLED display, WS2812 strip & a buzzer
```

Comment:
Just a heads up, this isn't actually used, unfortunately.

Reply:
Shall I remove it?